Classes/Issues/Merge/GithubClient+Merge.swift

```diff
@@ -18,7 +18,7 @@ extension GithubClient {
         repo: String,
         number: Int,
         type: IssueMergeType,
-        error: @escaping () -> Void
+        completionHandler: @escaping (Bool) -> Void
```

Comment:
Can we add a label to this tuple, like `@escaping (success: Bool) -> Void`? That makes it a bit easier to understand what the `Bool` is referring to.

Reply:
@BasThomas yes, you can.

Reply:
Sorry for bad english 😅
Do you want to fix it yourself or so that I fix it?

Reply:
If you want to fix it, go for it!

Reply:
All is ready